pkg/cmd/testhelpers/cmd_test_helpers.go

```diff
@@ -151,35 +151,28 @@ func CreateTestJxHomeDir() (string, string, error) {
 	if err != nil {
 		return "", "", err
 	}
-	exists, err := util.FileExists(path.Join(originalDir, "gitAuth.yaml"))
+	newDir, err := ioutil.TempDir("", ".jx")
 	if err != nil {
 		return "", "", err
 	}
-	if exists {
-		newDir, err := ioutil.TempDir("", ".jx")
-		if err != nil {
-			return "", "", err
-		}
-		contents, err := ioutil.ReadDir(originalDir)
-		if err != nil {
-			return "", "", err
-		}
-		for _, f := range contents {
-			if strings.HasSuffix(f.Name(), ".yaml") {
-				err = util.CopyFileOrDir(path.Join(originalDir, f.Name()), path.Join(newDir, f.Name()), true)
-				if err != nil {
-					return "", "", err
-				}
+	contents, err := ioutil.ReadDir(originalDir)
+	if err != nil {
+		return "", "", err
+	}
+	for _, f := range contents {
+		if strings.HasSuffix(f.Name(), ".yaml") {
+			err = util.CopyFileOrDir(path.Join(originalDir, f.Name()), path.Join(newDir, f.Name()), true)
+			if err != nil {
+				return "", "", err
 			}
 		}
-		err = os.Setenv("JX_HOME", newDir)
-		if err != nil {
-			os.Unsetenv("JX_HOME")
-			return "", "", err
-		}
-		return originalDir, newDir, nil
 	}
-	return "", "", nil
+	err = os.Setenv("JX_HOME", newDir)
+	if err != nil {
+		os.Unsetenv("JX_HOME")
```

Comment:
Technically it would probably be more accurate to call `os.Getenv`  prior to the call of `os.Setenv` to get the original value of JX_HOME and to reset to this value on error.